examples/starwars/lib/episode/hero_query.dart

```diff
@@ -36,15 +36,15 @@ class HeroForEpisode extends StatelessWidget {
         BoolCallback refetch,
         FetchMore fetchMore,
       }) {
-        if (result.errors != null) {
-          return Text(result.errors.toString());
-        }
-
         if (result.loading) {
-          return Center(
-            child: const CircularProgressIndicator(),
+          return const Center(
+            child: CircularProgressIndicator(),
           );
         }
+
```

Comment:
Would always check for errors before checking for loading. To abound infinite loading screens when actually an error had occurred.